wagtail/admin/templates/wagtailadmin/shared/headers/slim_header.html

```diff
@@ -9,12 +9,18 @@
     <div class="w-pl-slim-header sm:w-pl-5 w-min-h-slim-header sm:w-pr-2 w-w-full w-flex-1 w-overflow-x-auto w-box-border">
         <div class="w-flex w-flex-1 w-items-center w-overflow-hidden">
             {% block header_content %}
+                {% if breadcrumb_items %}
```

Comment:
Any chance we can rename this `breadrumbs_items` (plural breadcrumbs)? I see that this will not be a small change now with all the other PRs built on top.

Maybe that can come as a future PR before the release though.